src/ox/Reflection.java

```diff
@@ -91,7 +79,6 @@ public static void set(Object o, String fieldName, Object value) {
     }
     try {
       field.setAccessible(true);
-      modifiersField.setInt(field, field.getModifiers() & ~Modifier.FINAL);
```

Comment:
I think removing this may break some of our EZDB classes because some of our model classes (Bill, BillReview, BillDocument, GovNode, RuleNaics) have `final` member variables that EZDB sets using this library.

Have you tested this locally?

Reply:
yes its runs fine. 
only static final fields are affected by the restriction, according to the openjdk link

Reply:
ah OK